editor/icons/PrismMesh.svg

```diff
@@ -1 +1 @@
-<svg height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m7.9824 1.002a1.0001 1.0001 0 0 0 -.81445.44336l-5.9727 8.9609-.027344.03906a1 1 0 0 0 -.0625.10742 1 1 0 0 0 .44727 1.3418l6 3a1.0001 1.0001 0 0 0 .89453 0l6-3a1 1 0 0 0 .44726-1.3418 1 1 0 0 0 -.0625-.10742l-6-9a1.0001 1.0001 0 0 0 -.84961-.44336zm-.98242 4.3008v7.0801l-3.5391-1.7715zm2 0 3.5391 5.3086-3.5391 1.7715z" fill="#ffca5f"/></svg>
+<svg height="16" viewBox="0 0 14.999999 14.999999" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m3.7186288 1.8187229c-.3359407 1.0055883-1.843154 5.5147013-2.75469316 8.2751591-.14475589.524135.24550346 1.009193.72356176 1.172719 1.8418867.913065 3.6727531 1.85209 5.521487 2.749024.5482343.188028 1.0206745-.252238 1.4980384-.446285 1.6357702-.820591 3.2760662-1.632615 4.9090022-2.458571.499853-.285625.558497-.972259.247258-1.4174074-.925884-1.8413155-1.83539-3.6916611-2.771486-5.527346-.299268-.3976881-.82407-.5038792-1.2343075-.7530419-1.6256837-.8059034-4.405182-2.1931605-4.8731145-2.42664558s-.9298055-.17319355-1.2657462.83239478zm1.5157462 1.3785427 3.9316406 1.9648438-2.2128906 6.6406246-3.9316406-1.9648434zm5.251953 3.9335938 1.38086 2.7617187-2.7636724 1.3828129z" fill="#ffca5f"/></svg>
```

Comment:
The file should end with a newline character (`LF`).

If it hasn't been yet, you should also optimize it with `svgcleaner`: https://docs.godotengine.org/en/stable/development/editor/creating_icons.html#icon-optimization

Reply:
Added `LF` and seems to pass now. `svgcleaner` was actually the culprit here and removed the newline. There were also some weird scaling issues in Inkscape, I changed it so that `viewBox` is `0 0 16 16` again.